tests/e2e/utils/network_utils.go

```diff
@@ -127,14 +127,21 @@ func (azureTestClient *AzureTestClient) DeleteSubnet(vnetName string, subnetName
 			return false, nil
```

Comment:
Could you get the DELETE operation failure and log the delete error as well?

Reply:
Updated. PTAL.

Reply:
Updated. Used `%w`

Reply:
%w is used for wrapping an error into another one, not for logging. I think we can use %s for err.Error() instead.

Reply:
Makes sense. Used `%q` and `err.Error()` now.